realm/realm-library/src/syncIntegrationTest/java/io/realm/PermissionManagerTests.java

```diff
@@ -69,6 +70,12 @@ public void getPermissions_returnLoadedResults() {
             @Override
             public void onSuccess(RealmResults<Permission> permissions) {
                 assertTrue(permissions.isLoaded());
+                if (permissions.size() > 1) {
```

Comment:
There appears to be a timing issue. We have both a `__management` and `__permission` Realm and there is a timing issue for when the entries are being created in the `__permission` Realm. My implementation of the logic is wrong, and right now I'm trying to wait for both the `__permission` and `__management` Realm to appear, but for reasons, I don't fully understand yet,  one of these entries are never created.